stable/object-bucket-claim/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v2
+name: object-bucket-claim
+description: A Helm chart for ObjectBucketClaim to be consumed by other applications. CRD is included to allow testing only.
+home: https://github.com/mesosphere/charts
+keywords:
+  - kommander
+version: 0.1.0
+appVersion: "1.16.0"
```

Comment:
i guess techinically the CR objectbucketclaim is not really bound to of ceph version so maybe we can use 0.1.0 here.

Reply:
done